extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
Just to make sure, this should be exactly the same with https://github.com/ampproject/amphtml/pull/38179/files? If not, could you cope it directly to here?